.github/workflows/build.yaml

```diff
@@ -24,6 +42,9 @@ jobs:
           - runtime: windows
             os: windows-latest
             reportName: windows-test-report
+    env:
+      USE_LOCAL_PLUGIN: ${{ inputs.useLocalPlugin || 'false' }}
```

Comment:
Why is the default value given is a string even though useLocalPlugin is a boolean type?

Reply:
Changed it to boolean